dev/addons/admin-panel-values.yml

```diff
@@ -0,0 +1,3 @@
+adminPanel:
+  graphqlUrl: http://api:4002/graphql
```

Comment:
Needed to change this to
`http://localhost:4002/graphql`
to be abale to pass the captcha on the admin panel.
(as in the readme forwarded the ports 4002 (http ingress), 3000 (web-wallet) and 3001 (admin panel))

able to log in, but when querying a phone number in the admin panel getting a 400 error:
![image](https://user-images.githubusercontent.com/43343391/219459397-d1566920-17eb-4280-a1fa-c18f48dda349.png)

Reply:
removed admin panel stuff for a follow up PR